.github/workflows/cd.yml

```diff
@@ -0,0 +1,44 @@
+name: Dependabot CD to GCP
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  deploy-dependabot-merge-action-app:
+    name: Build & Deploy dependabot-merge-action-app
+    runs-on: ubuntu-latest
+
+    env:
+      PROJECT_ID: ${{ secrets.GCP_PROJECT_ID }}
+      CLOUDRUN_SERVICE_NAME: dependabot-merge-action-app
+      CLOUDRUN_SERVICE_REGION: us-central1
+      API_ID_VALUE: ${{ secrets.API_ID }}
+      PRIVATE_KEY_VALUE: ${{ secrets.PRIVATE_KEY}}
+
+    steps:
+      - name: Checkout project
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+      - name: Set up gcloud CLI
+        uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+        with:
+          version: latest
```

Comment:
let's omit this as it's the default value

Reply:
Removed the value